Kickstarter-iOS/Features/ConfirmDetails/Controllers/PostCampaignPledgeSummaryViewController.swift

```diff
@@ -0,0 +1,171 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class PostCampaignPledgeSummaryViewController: UIViewController {
```

Comment:
I don't understand why we need a separate post campaign version of this class. The styling looks the same. If it's just hiding the "If the project reaches its funding goal" string that's the difference, we should just add an option to show/hide that instead of making a separate copy of the class. If there are other notable changes between this file and the standard pledge summary, can you call them out?

Reply:
Yea there aren't more notable differences. I mostly wanted to maintain context and keep all of the PCP related components isolated so that removing the old/current flow would be easier

Reply:
I don't think we'd ever be able to remove the old version, though; the "If the project reaches its funding goal" text is still going to be relevant in the normal pledge flow, even if we do update the rest of the styling to match the PCP flow. So unless we want to keep two almost identical versions of this class in sync forever (which I really don't), we should keep the old class and instead modify it to make it support both cases.